build/yamls/base/grafana.yml

```diff
@@ -107,7 +107,7 @@ spec:
           imagePullPolicy: IfNotPresent
           env:
             - name: GF_INSTALL_PLUGINS
-              value: "https://downloads.antrea.io/artifacts/grafana-custom-plugins/theia-grafana-sankey-plugin-1.0.0.zip;theia-grafana-sankey-plugin,grafana-clickhouse-datasource 1.0.1"
+              value: "https://downloads.antrea.io/artifacts/grafana-custom-plugins/theia-grafana-sankey-plugin-1.0.0.zip;theia-grafana-sankey-plugin,https://github.com/heanlan/grafana-chord-plugin/releases/download/1.0.0/grafana-chord-plugin-1.0.0.zip;theia-grafana-chord-plugin,grafana-clickhouse-datasource 1.0.1"
```

Comment:
Shall we update this before we merge the PR?

Reply:
all the links are updated